sdk/translation/azure-ai-translation-document/CHANGELOG.md

```diff
@@ -4,6 +4,17 @@
 
 This version of the SDK defaults to the latest supported service version, which currently is v1.0
 
+**Breaking changes**
+
+- `create_translation_job` was renamed to `begin_translation` and now returns a `DocumentTranslationPoller`
```

Comment:
would it be better to say that the method was removed and a new one that follows a different approach (LRO) was added?

Reply:
yeah I like that better, will update